### PR TITLE
Fix dynamic imports

### PR DESCRIPTION
**Pull request by kbandla**
This is a fix for #61 and #141. We now make sure that modules-to-import are searched 'relative to the directory of the module'.

### CODE REVIEW

File: dpkt/ethernet.py [hunk 1]
```diff
@@ -124,10 +124,10 @@ def __load_types():
             name = k[9:]
             modname = name.lower()
             try:
-                mod = __import__(modname, g)
-            except ImportError:
+                mod = __import__(modname, g, level=0)
```
Comment (sunhao2014) [inline dpkt/ethernet.py]: I wonder why in `ethernet.py` the modification is `level=0` while other modules are all `level=1`.

Comment (kbandla) [inline dpkt/ethernet.py]: good catch. Fix coming up shortly

**Comment by kbandla** [comment]
please review and merge if it looks good

**Comment by kbandla** [comment]
Looks like its working, based on #141. Merging.

**Pull request merged**